js/mercado.js

```diff
@@ -151,7 +187,7 @@ module.exports = class mercado extends Exchange {
     async fetchBalance (params = {}) {
         let response = await this.privatePostGetAccountInfo ();
         let balances = response['response_data']['balance'];
-        let result = { 'info': response };
+        let result = {};
```

Comment:
This line has to be `const result = { 'info': response }` (we must include the `response` in the `info` field inside the balance or any other response).